lib/parallel_tests/cli.rb

```diff
@@ -67,6 +68,12 @@ def run_tests_in_parallel(num_processes, options)
       abort final_fail_message if any_test_failed?(test_results)
     end
 
+    def finish_hook(options)
```

Comment:
could just inline that ... `finish: -> (_, i, _) { ParallelTests.pids.delete(i) }`